src/components/rating/Star.jsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import Icon, {types, colors} from '../icons/Icon';
+import classnames from 'classnames';
+import PropTypes from 'prop-types';
+
+const Star = ({size, checked, onClick = () => undefined}) => {
```

Comment:
If you just showing rating you don't need specify function. IMHO having default dummy function is better than making checking if there is function inside